test/integration/admin_login_test.rb

```diff
@@ -13,9 +14,9 @@ def setup
                 email: 'example@example.com',
                 role: 1)
 
-    @item1 = Item.create(title: 'espresso', price: 9000)
+    @item1 = Item.create(title: 'espresso', description: "this is black gold", price: 30000)
     @category1 = item1.categories.create(name: 'Hot Beverages')
-    @item2 = Item.create(title: 'cold pressed coffee', price: 8000)
+    @item2 = Item.create(title: 'cold pressed coffee', price: 8000, description: "hipster nonsense", price: 20000)
```

Comment:
Line is too long. [114/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.